definition/definition.go

```diff
@@ -14,17 +14,18 @@ import (
 
 // Definition ...
 type Definition struct {
-	Name           string          `json:"name"`
-	Datacenter     string          `json:"datacenter"`
-	ErnestIP       []string        `json:"ernest_ip"`
-	ServiceIP      string          `json:"service_ip"`
-	VpcID          string          `json:"vpc_id"`
-	VpcSubnet      string          `json:"vpc_subnet"`
-	Networks       []Network       `json:"networks"`
-	Instances      []Instance      `json:"instances"`
-	SecurityGroups []SecurityGroup `json:"security_groups"`
-	ELBs           []ELB           `json:"loadbalancers"`
-	NatGateways    []NatGateway    `json:"nat_gateways"`
+	Name              string          `json:"name"`
+	Datacenter        string          `json:"datacenter"`
+	ErnestIP          []string        `json:"ernest_ip"`
+	ServiceIP         string          `json:"service_ip"`
+	VpcID             string          `json:"vpc_id"`
+	VpcSubnet         string          `json:"vpc_subnet"`
+	Networks          []Network       `json:"networks"`
+	Instances         []Instance      `json:"instances"`
+	SecurityGroups    []SecurityGroup `json:"security_groups"`
+	ELBs              []ELB           `json:"loadbalancers"`
+	NatGateways       []NatGateway    `json:"nat_gateways"`
+	DatacenterDetails Datacenter      `json:"-"`
```

Comment:
It's not actually overridable on the yaml, as the json tag is "-", it's just used to validate the definition against additional daracenter values